lang/src/test/kotlin/org/partiql/lang/eval/evaluatortestframework/ExprValueStrictEquals.kt

```diff
@@ -14,17 +14,26 @@ import org.partiql.lang.eval.timeValue
 import org.partiql.lang.eval.timestampValue
 
 /**
- * Only used to verify the actual `ExprValue` is the same as the expected `ExprValue` in test
+ * Strict equality that captures equality in the PartQL data model. That is, for 2 scalars, they are considered as equal
+ * if they have the same type & value. For 2 collections, each element in one collection must be strictly equal to
+ * an element in the other collection. The equality of collections respects elements order for sexp & list, and not for
+ * bag & struct.
  *
- * Ensures 2 [ExprValue]s to have the same type & value
+ * Note that, the equality we define here is different from [ExprValue.exprEquals]. That one is used for PartiQL equals
+ * operator & ORDER BY clause. The equality there ignores data types in some cases, e.g., the partiql expressions
+ * `1.0 = 1` returns true (`1.0` is a decimal, while `1` is an integer).
+ *
```

Comment:
"ignores data types in some cases" --> "equates values of distinct data types in some cases"

And, as we are here, a few more nits I'd might let pass otherwise: 

- & on line 23 is particularly egregious; do "and" at least here; 
- exprEquals is not used in ORDER BY -- a comparison consistent with it is what is used there; I'd reword as "... different from [ExprValue.exprEquals], which is based on the comparison in PartiQL ORDER BY clause. The comparison there ignores/equates ..."  
- "the partiql expressions `1.0 = 1`" --> "the PartiQL expression `1.0 = 1`"
- ... --> "even though 1.0 is a decimal"